engine/src/simulation.cpp

```diff
@@ -1212,6 +1262,24 @@ SimulationResult Simulation::run() {
 
   // Abort handler:
   SimulationMachine machine;
+
+  reset_fn_ = [this, &ctx, &machine]() {
+    // static size_t requests = 0;
+
+    logger()->info(
+        R"(Match signal quit(like CTRL+'\') to reset functionality.)");
+
+    // requests += 1;
```

Comment:
I see why you may want to use `SIGQUIT`. It's so you can just use `Ctrl+\`.

- [ ] Is there another way to achieve this without clobbering already used signals?

Reply:
yes, override system signal is not a good solution.

Reply:
I should extend SIGINT to be interactive (and maybe provide an option).
Hypothetical example:
```
II 15:00:08.658 [cloe] Signal caught.
What do you want to do [abort|restart]? r
II 15:00:08.659 [cloe] Restarting simulation...
```